animations-dbus/animations-dbus-server-object.c

```diff
@@ -735,3 +744,68 @@ animations_dbus_server_new_with_connection_async (AnimationsDbusServerEffectFact
                               "effect-factory", factory,
                               NULL);
 }
+
+/**
+ * animations_dbus_server_stop:
+ * @self: an #AnimationsDbusServer
+ * @cancellable: (nullable): a #GCancellable, or %NULL
+ * @error: return location for a #GError, or %NULL
+ *
+ * Stop the server, unexport all D-Bus objects, and unown the well-known
+ * D-Bus name for the service.
+ *
+ * Returns: %TRUE on success, %FALSE otherwise
+ */
+gboolean
+animations_dbus_server_stop (AnimationsDbusServer  *self,
+                             GCancellable          *cancellable,
+                             GError               **error)
+{
+  AnimationsDbusServerPrivate *priv = animations_dbus_server_get_instance_private (self);
+  guint i;
+  GHashTableIter iter;
```

Comment:
I guess we could declare `iter` together with `key` in the conditional below.

Reply:
Fixed.